packages/nimble-components/src/wafer-map/modules/prerendering.ts

```diff
@@ -130,21 +134,33 @@ export class Prerendering {
     }
 
     private calculateFillStyle(
-        die: WaferMapDie,
-        colorScaleMode: WaferMapColorScaleMode
+        value: string,
+        colorScaleMode: WaferMapColorScaleMode,
+        highlightedValues: Readonly<string[]>
```

Comment:
Should this use `ReadonlyArray<string>`?

Reply:
changing it triggers this eslint error "Array type using 'ReadonlyArray<string>' is forbidden. Use 'readonly string[]' instead.". Should I suppress it for this input?

Reply:
nope, linter got it